sessions/css-animations/toast-animations/README.md

```diff
@@ -0,0 +1,24 @@
+# Toast Animation
+
+In this codesandbox you are gonna practice to animate properties in CSS!
+
+## Task
+
+### Part 1
+
+The hover effect on the submit button looks quite ugly. Let's change that.
+
+- Add a `transition` property to the submit button and animate the transition to the hover state for all properties. Find some values that you personally like.
+
+### Part 2
+
+When you click on the button, a toast appears on the bottom right corner. Lets animate it as well, but this time we want a different animation when the toast leaves the screen!
+
+- first define a transition in the `.toast.enter` ruleset. This will be describing the enter animation.
+- then define the exit animation in the `.toast.exit` ruleset. This time use the bezier tool in the developer tools to find an animation that overshoots a little to the left before the toast leaves to the right.
```

Comment:
```suggestion
- then define the exit animation in the `.toast.exit` ruleset. This time use the bezier tool in the developer tools to find an animation that bounces a little to the left before the toast leaves to the right.
```
I'm going to be crazy nit-picky here, but "overshoots" isn't the correct term here. It would be the correct term if it was on the `.toast.enter` transition. This animation type is usually referred to as a bounce animation, which is fairly similar to a rubber band animation. :)